setup.cfg

```diff
@@ -30,6 +30,7 @@ disable=abstract-method, # raises false positives in beam
 
 [FORMAT]
 # Allow some single and two letter variables, but keep overall invalid-name rule
-good-names=i,j,k,f,ex,e,_,p,ip,cm
+# Also allow censoredplanet-analysis as a module name
+good-names=i,j,k,f,ex,e,_,p,ip,cm,censoredplanet-analysis
```

Comment:
Not used, revert?

Also, prefer underscores: `censoredplanet_analysis`, otherwise you don't have a valid symbol.

Reply:
This is to fix a error where it started lint triggering on the github repo name `censoredplanet-analysis`. It's too late to change that, so we'll just have to add an exception for the name.